### PR TITLE
Rhin 3031/update packages

### PR DESCRIPTION
**Pull request by adlondon**
✔︎ I've tested in all browsers (IE11, Firefox, Safari, Chrome, and iOS)

### What should this PR do?
Updates packages to latest

### CODE REVIEW

File: config/server.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ import browserSync from 'browser-sync';
  * @return {void}
  */
 export default function server() {
+  console.log('HEREEEREREE');
```
Comment (juanfabrega) [inline config/server.js]: Can we get rid of this one?

File: config/watch.js [hunk 2]
```diff
@@ -9,11 +9,13 @@ const { reload } = browserSync;
  * Watch for changes in files
  * @return {void}
  */
-export default function watch() {
-  gulp.watch(paths.svg.src, ['icons', reload]);
-  gulp.watch(paths.styles.docAll, ['dist:styles', 'docs:styles', reload]);
-  gulp.watch(paths.scripts.src, ['dist:scripts', 'docs:scripts', reload]);
-  gulp.watch([paths.metalsmith.pages, paths.metalsmith.templates], ['docs:site', reload]);
+export default function watch(cb) {
+  console.log('WHATCHHHHH');
```
Comment (juanfabrega) [inline config/watch.js]: remove console.log

File: config/webpack.dist.config.js [hunk 3]
```diff
@@ -52,12 +53,6 @@ module.exports = {
     new webpack.DefinePlugin({
       'process.env': { NODE_ENV: JSON.stringify(nodeEnv) },
     }),
-    // Disabling until https://github.com/fritz-c/react-image-lightbox/issues/88 is resolved
-    // Causing an issue within main application
-    // ***** 2/28/2019 UPDATE *****
-    // Re-enabling UglifyJs since conflict seems to be resolved. Keeping original comment above in case issue re-surfaces. -- Juan Fabrega
-    new webpack.optimize.UglifyJsPlugin(),
```
Comment (juanfabrega) [inline config/webpack.dist.config.js]: Is this being handled somewhere else now?

File: dist/scripts/NotificationSystem/actions/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+"use strict";
```
Comment (juanfabrega) [inline dist/scripts/NotificationSystem/actions/index.js]: Looks like stuff isn't getting bundled anymore. All these scripts should be under `rhinostyle.min.js` and `rhinostyle.min.js.map`. There should be no source code in the `dist` folder.

File: config/webpack.dist.config.js (hunk 3 above)
Comment (adlondon) [reply config/webpack.dist.config.js]: handled now with terser-webpack-plugin

**Pull request merged**